test/components/xDragBoxLayerTests.ts

```diff
@@ -1,178 +1,155 @@
-
-      let boundsAfter = dbl.bounds();
-      assert.strictEqual(boundsAfter.topLeft.x, boundsBefore.topLeft.x, "box keeps same left edge");
-      assert.strictEqual(boundsAfter.topLeft.y, 0, "box still starts at top");
-      assert.strictEqual(boundsAfter.bottomRight.x, boundsBefore.bottomRight.x, "box keeps same right edge");
-      assert.strictEqual(boundsAfter.bottomRight.y, dbl.height(), "box still ends at bottom");
-      svg.remove();
-    });
+      it("has the correct bounds()", () => {
+        dbl.boxVisible(true);
+        dbl.renderTo(svg);
 
-    it("throws error on getting y scale", () => {
-      let dbl = new Plottable.Components.XDragBoxLayer();
-      assert.throws(() => dbl.yScale(), "no yScale");
-    });
+        dbl.bounds({
+          topLeft: quarterTopLeftPoint,
+          bottomRight: middlePoint
+        });
 
-    it("throws error on setting y scale", () => {
-      let dbl = new Plottable.Components.XDragBoxLayer();
-      assert.throws(() => dbl.yScale(new Plottable.Scales.Linear()), "yScales cannot be set");
-    });
+        let actualBounds = dbl.bounds();
+        assert.strictEqual(actualBounds.topLeft.y, 0, "box starts at top");
+        assert.strictEqual(actualBounds.topLeft.x, quarterTopLeftPoint.x, "left edge set correctly");
+        assert.strictEqual(actualBounds.bottomRight.y, dbl.height(), "box ends at bottom");
+        assert.strictEqual(actualBounds.bottomRight.x, middlePoint.x, "right edge set correctly");
 
-    it("throws error on getting y extent", () => {
-      let dbl = new Plottable.Components.XDragBoxLayer();
-      assert.throws(() => dbl.yExtent(), "no yExtent");
-    });
+        svg.remove();
+      });
 
-    it("moves only in x", () => {
-      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
-      let dbl = new Plottable.Components.XDragBoxLayer();
-      dbl.boxVisible(true);
-      dbl.movable(true);
-      dbl.renderTo(svg);
+      it("resizes only in x", () => {
+        dbl.boxVisible(true);
+        dbl.resizable(true);
+        dbl.renderTo(svg);
+
+        dbl.bounds({
+          topLeft: quarterTopLeftPoint,
+          bottomRight: middlePoint
+        });
+
+        let actualBounds = dbl.bounds();
+        let dragTo = {
+          x: SVG_WIDTH * 3 / 4,
+          y: SVG_HEIGHT / 2
+        };
+        TestMethods.triggerFakeDragSequence(dbl.background(), actualBounds.bottomRight, dragTo);
+        actualBounds = dbl.bounds();
+        assert.strictEqual(actualBounds.bottomRight.x, dragTo.x, "resized in x");
+        assert.strictEqual(actualBounds.topLeft.y, 0, "box still starts at top");
+        assert.strictEqual(actualBounds.bottomRight.y, dbl.height(), "box still ends at bottom");
+        svg.remove();
+      });
 
-      let topLeft = {
-        x: SVG_WIDTH / 4,
-        y: SVG_HEIGHT / 4
-      };
-      let bottomRight = {
-        x: SVG_WIDTH * 3 / 4,
-        y: SVG_HEIGHT * 3 / 4
-      };
+      it("stays full height after resizing", () => {
```

Comment:
Probably rename this to distinguish it from the previous test -- maybe something like `"stays full-height when redrawn at a different size"`?